### PR TITLE
Pract. 07 (References): Add 07-09 | Georgi Baltiev

### PR DESCRIPTION
**Pull request by georgibaltiev**
Georgi Baltiev
2 2MI0600168

### CODE REVIEW

File: Task07.cpp [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+int main() {
+    int n, hour = 0, minute = 0;
+    std::cin >> n;
+    if(n < 0) return 0;
+    for(int i = 0; i < n; i++) {
+        int startingHour, startingMinute, duration;
+        std::cin >> startingHour >> startingMinute >> duration;
+        calculateHours(hour, minute, startingHour, startingMinute, duration);
+    }
+    std::cout << hour << " " << minute << std::endl;
```
Comment (GeorgiTerziev02) [inline Task07.cpp:28]: You do not check if hour is >= 24 when printing
Try the following input
4
19 15 120
20 20 25
18 25 230
21 10 3232

File: Task07.cpp [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        startHour++;
+    }
+
+    if(startHour > hour) {
+        hour = startHour;
+        minute = startMinute;
+    }
+    else if(startHour == hour) {
+        if(startMinute > minute) minute = startMinute;
+    }
```
Comment (GeorgiTerziev02) [inline Task07.cpp:16]: else if(.... && .....)

**Review by GeorgiTerziev02 — CHANGES_REQUESTED** [review]

**Review by GeorgiTerziev02 — APPROVED** [review]

**Comment by Justsvetoslavov** [comment]
@GeorgiTerziev02 @georgibaltiev Is it me or the task are not in the correct directory?

**Review by Justsvetoslavov — CHANGES_REQUESTED** [review]
Waiting for a reply on the above comment

**Pull request closed without merging**